src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java

```diff
@@ -24,70 +23,73 @@ public ArrebolJobSynchronizer(Properties properties) {
 	@Override
 	public JDFJob synchronizeJob(JDFJob job) {
 		try {
-			String arrebolJobJson = this.requestsHelper.getJobJSON(job.getJobIdArrebol());
-			LOGGER.debug("JSON Response [" + arrebolJobJson + "]");
-			Set<ArrebolTaskState> taskStates = this.getJobTaskStates(arrebolJobJson);
-			JDFJobState jobState = getJobState(taskStates);
-			LOGGER.debug("Tasks states set [" + taskStates.toString() + "] resuming to State [" + jobState.value() + "]");
-			job.setState(jobState);
+			String arrebolJobId = job.getJobIdArrebol();
+			if(arrebolJobId != null){
+				String arrebolJobJson = this.requestsHelper.getJobJSON(arrebolJobId);
+				LOGGER.debug("JSON Response [" + arrebolJobJson + "]");
+
+				Gson gson = new Gson();
+				ArrebolJob arrebolJob = gson.fromJson(arrebolJobJson, ArrebolJob.class);
+				this.updateJob(job, arrebolJob);
+			} else {
+				LOGGER.error("ArrebolJobId from Job [" + job.getId() + "] is null.");
+			}
```

Comment:
always add the throwable as arg to the LOGGER. error method

Reply:
Ok. Done.